### DISCUSSION TITLE
希望增加选项

### DISCUSSION THREAD

**Discussion by ZokuTe** [alex8088/electron-vite#200] [Ideas]
### Clear and concise description of the problem

eslint和prettier在我的工作流中很少出现，代码的格式化与检查都是由idea完成的，而idea的格式化和检查会被插件所覆盖，极大影响了我的使用习惯，每次进行手动卸载又不是显得便利。

### Suggested solution

希望让eslint和prettier成为可选项目

### Validations

- [X] Follow the [Code of Conduct](https://github.com/alex8088/electron-vite/blob/master/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/alex8088/electron-vite/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://evite.netlify.app).
- [X] Check that there isn't [already an issue](https://github.com/alex8088/electron-vite/issues) that reports the same bug to avoid creating a duplicate.

**Comment by liuyike98** [comment]
是的,我也是用的编辑器自带的格式化,每次都要手动卸载删文件,我觉得可以考虑一下

**Comment by rico0808** [comment]
我觉得保留是最好的，eslint 和 prettier 对于团队是很好的代码规范工具。
